### PR TITLE
Add prom/mysqld-exporter to mysqlha

### PR DESCRIPTION
**Pull request by Aisuko**
Signed-off-by: Aisuko <urakiny@gmail.com>

#### What this PR does / why we need it:
This PR add `prom/mysqld-exporter`( the Prometheus exporter for MySQL server metrics) to the chart `incubator/mysqlha`  for every pod.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: incubator/mysqlha/templates/svc.yaml [hunk 1]
```diff
@@ -31,5 +31,7 @@ spec:
   ports:
   - name: {{ template "fullname" . }}
     port: 3306
+  - name: metrics
```
Comment (davidkarlsen) [inline incubator/mysqlha/templates/svc.yaml:39]: should it be possible to add annotations to the service? This is typically used to make prometheus aware of the endpoints and scrape them

Comment (Aisuko) [reply incubator/mysqlha/templates/svc.yaml:39]: Add metrics annotations to the service.

```
  annotations:
{{- if and (.Values.metrics.enabled) (.Values.metrics.annotations) }}
{{ toYaml .Values.metrics.annotations | indent 4 }}
{{- end }}
```

**Pull request merged**